<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trip</class>
 <widget class="QMainWindow" name="trip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #003366;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1051</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 25px;</string>
      </property>
      <property name="text">
       <string>Purchase Souvenirs</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tripButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>650</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:DarkGrey ;
color: white;
border-radius: 10px;
}
QPushButton:hover:!pressed
{
background: Gray;
color: white;
border-radius: 10px;
}color: white;</string>
      </property>
      <property name="text">
       <string>Next Team</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>150</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Cart</string>
      </property>
     </widget>
     <widget class="QLabel" name="tripTeamName">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>50</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 35px;
color: white;</string>
      </property>
      <property name="text">
       <string>TEAM NAME</string>
      </property>
     </widget>
     <widget class="QTableView" name="souvenirsTable">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>61</y>
        <width>671</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="cartTable">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>180</y>
        <width>341</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 35px;</string>
      </property>
      <property name="text">
       <string>Trip Review</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>85</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Teams visited:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="arenaTable">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>120</y>
        <width>256</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>90</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Arenas Visited:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="reviewSouvenirTable">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>120</y>
        <width>256</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>90</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Souvenirs purchased:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="reviewTeamTable">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>256</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>490</y>
        <width>221</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="title">
       <string>Totals</string>
      </property>
      <widget class="QLabel" name="totalDistanceNum">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="totalDistanceLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Total Distance</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Total Spent</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Total Arenas</string>
       </property>
      </widget>
      <widget class="QLabel" name="totalSpent">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="totalArenasVisited">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>451</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelName">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 35px;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
